### ISSUE TITLE
IncomingTransfer doesn't return subaddr index

### ISSUE THREAD

**Issue by t-900-a** [MarinX/monerorpc#1]
**Describe the bug**
When calling 

`client.Wallet.IncomingTransfers(&incomingTransfersReq)`

the error is returned

`json: cannot unmarshal object into Go struct field IncomingTransfer.transfers.subaddr_index of type uint64`

**To Reproduce**
https://github.com/t-900-a/fider-monero-bot/blob/b85886d9e1bc26e95f10f35977d505bb64553e73/process-monero/main.go#L82

**Expected behavior**
subaddresses should be parsed correctly

**Go Version**
Please type your golang version
1.16.5

**Monerod Version**
Please type your monerod version
17.2.0

**Comment by t-900-a** [comment]
More details after further investigation:
subaddr_index is not of type uint64, rather "major" & "minor" are.

```
      "subaddr_index": {
        "major": 0,
        "minor": 32
      }
```

Struct `Index ` is already defined. Expect a pull request with subaddr_index type changed to index similar to the Payment struct.

https://github.com/MarinX/monerorpc/blob/1dc1175f645cfa06f7c6e42bbaa49a3af34a3c78/wallet/models.go#L545

**Comment by MarinX** [comment]
Solved by PR #2
